config/locales/en.yml

```diff
@@ -41,6 +41,8 @@ en:
       header: 'Edit your Team'
       legend: 'New Team Information'
       note: 'All fields are required.'
```

Comment:
You should be able to remove teams.edit.note because it's not used anywhere else